File: 251003/Habrus/src/main/java/by/bsuir/distcomp/entity/Article.java
package by.bsuir.distcomp.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Article implements Identifiable {

    private Long id;
    private Long authorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;

}

File: 251002/Nikonorova/lab1_rest/Models/Article.cs
using System.Text.Json.Serialization;

namespace distcomp.Models
{
    public class Article
    {
        [JsonPropertyName("id")]
        public long Id { get; set; }
        [JsonPropertyName("creatorId")]
        public long CreatorId { get; set; }
        [JsonPropertyName("title")]
        public string Title { get; set; }
        [JsonPropertyName("content")]
        public string Content { get; set; }
        [JsonPropertyName("created")]
        public DateTime Created { get; set; }
        [JsonPropertyName("modified")]
        public DateTime Modified { get; set; }
    }
}
